cartography/intel/aws/ec2/images.py

```diff
@@ -19,7 +19,7 @@
 def get_images(boto3_session: boto3.session.Session, region: str) -> List[Dict]:
     client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
     try:
-        images = client.describe_images(Owners=['self'])['Images']
```

Comment:
why remove thtis?

Reply:
Looks like I overlooked this during the PR merge conflicts. Rectified it. My bad. Apologies.

Reply:
Good you pointed this out. I would have undone a fix 🙈